api/web3server.go

```diff
@@ -982,6 +984,26 @@ func (svr *web3Handler) unsubscribe(in *gjson.Result) (interface{}, error) {
 	return chainListener.RemoveResponder(id.String())
 }
 
+func (svr *web3Handler) getBlobSidecars(in *gjson.Result) (interface{}, error) {
+	blkNum := in.Get("params.0")
+	if !blkNum.Exists() {
+		return nil, errInvalidFormat
+	}
+	num, err := svr.parseBlockNumber(blkNum.String())
+	if err != nil {
+		return nil, err
+	}
+	res, err := svr.coreService.BlobSidecarsByHeight(num)
+	switch errors.Cause(err) {
+	case nil:
+		return res, nil
+	case ErrNotFound:
+		return nil, nil
```

Comment:
`ErrNotFound` should return nil to caller?

Reply:
yeah, just like getBlock and getTransaction